### PR TITLE
Add CKAN Google Analytics token

### PR DESCRIPTION
**Pull request by emmabeynon**
For https://trello.com/c/vXAA8ulQ/412-site-analytics-gives-404

### CODE REVIEW

File: modules/govuk/templates/ckan/ckan.ini.erb [hunk 1]
```diff
@@ -151,7 +151,7 @@ ckan.max_resource_size = 50
 ## GA Settings
 googleanalytics.account = data.gov.uk
 googleanalytics.id = UA-10855508-1
-googleanalytics.token.filepath = /var/apps/ckan/ga_token
+googleanalytics.token.filepath = /var/ckan/ga_token
```
[the remark this answers is not in the document]
Comment (emmabeynon) [reply modules/govuk/templates/ckan/ckan.ini.erb]: Now done :)

File: modules/govuk/manifests/apps/ckan.pp [hunk 2]
```diff
@@ -147,6 +150,14 @@
       notify  => Service['ckan'],
     } ->
 
+    file { "${ckan_home}/ga_token.json":
```
Comment (andrewgarner) [inline modules/govuk/manifests/apps/ckan.pp]: This should be `"${ckan_home}/google_analytics_token.json"`

File: modules/govuk/manifests/apps/ckan.pp [hunk 3]
```diff
@@ -147,6 +150,14 @@
...
+      ensure  => file,
+      content => template('govuk/ckan/ga_token.json.erb'),
```
Comment (andrewgarner) [inline modules/govuk/manifests/apps/ckan.pp]: This should be `template('govuk/ckan/google_analytics_token.json.erb'),`

File: hieradata/vagrant_credentials.yaml [hunk 4]
```diff
@@ -26,6 +26,9 @@ collectd::plugin::postgresql::password: 'password'
 govuk::apps::bouncer::postgresql_role::password: 'sKnvsmsAEeviNNl3Ded90NzxoT7eFI6Qd1cOVmaA'
 govuk::apps::ckan::db::password: 'goh8Deev3peki7liuxool7mo'
 govuk::apps::ckan::db_password: "%{hiera('govuk::apps::ckan::db::password')}"
+govuk::apps::ckan::google_analytics_access_token: "%{hiera('govuk::apps::ckan::db::google_analytics_access_token')}"
```
Comment (andrewgarner) [inline hieradata/vagrant_credentials.yaml]: Have the secrets definitely been added to the `govuk::apps::ckan::db` (database) namespace?

**Comment by emmabeynon** [comment]
@andrewgarner these should be sorted now

**Pull request merged**